plugins/inputs/docker/docker.go

```diff
@@ -134,6 +148,15 @@ func (d *Docker) Gather(acc telegraf.Accumulator) error {
 		d.client = c
 	}
 
+	// Create label filters
+	if len(d.LabelInclude) != 0 {
```

Comment:
We should only have it compile once, you could use a init function guarded with a bool.